iBurn/Images.xcassets/pin_centerCamp.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "center_camp.png",
```

Comment:
Lol I guess this icon no longer makes sense

Reply:
Lol yeah, no coffee this year